index.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var app = express();
+var bodyParser = require('body-parser');
+var compliments = require("./models/compliment")
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+app.set("view engine","hbs");
+app.use(express.static(__dirname + "/public"));
+
+function getRandomCompliments(module){
+  var randomIndex = Math.floor((Math.random() * module.length));
+  var randomInstance = module[randomIndex];
+  return randomInstance;
+}
+
+app.get("/", function(req, res){
+  var randomCompliment = getRandomCompliments(compliments);
+  res.render("index.hbs", {compliment: randomCompliment});
```

Comment:
could pass in a random color via a similar process that you did with generating a random compliment